hw/top_earlgrey/cdc/cdc_waivers.misc.tcl

```diff
@@ -4,76 +4,106 @@
 #
 # Verix CDC waiver file
```

Comment:
Another question (sorry I don't remember): why were these waivers put into the `*.misc.tcl` file instead of the `*.w_data.tcl` and `*.w_masync.tcl` ones?

Reply:
Discussed with @jeoongp offline. This is an artifact of the recent restructuring into different files. We decided to leave this for now. @jeoongp is going to clean the structure up a bit more post M2.

Reply:
Yes, misc items will be reviewed and added onto w_data and w_masync waivers after M2.